semantic-typescript/src/Language/TypeScript.hs

```diff
@@ -2,15 +2,15 @@
 -- | Semantic functionality for TypeScript programs.
 module Language.TypeScript
 ( Term(..)
-, TreeSitter.TypeScript.tree_sitter_typescript
+, Language.TypeScript.Grammar.tree_sitter_typescript
```

Comment:
This seems to be erroring:

```
[ 8 of 10] Compiling Language.TypeScript.AST ( semantic-typescript/src/Language/TypeScript/AST.hs, dist-newstyle/build/x86_64-osx/ghc-8.8.1/build-repl/Language/TypeScript/AST.o )

semantic-typescript/src/Language/TypeScript/AST.hs:19:1: error:
    Could not find module ‘Language.TypeScript.Grammar’
    Perhaps you meant
      Language.TypeScript.Syntax
      Language.TypeScript.Tags
      Language.TypeScript.Term
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
19 | import qualified Language.TypeScript.Grammar as Grammar
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Failed, 7 modules loaded.
```

Reply:
Is that still the case? It looks like CI has succeeded?